src/mixins/userStatus.js

```diff
@@ -54,7 +54,7 @@ export default {
 			}
 
 			try {
-				const { data } = await axios.get(generateOcsUrl('apps/user_status/api/v1', 2) + `statuses/${encodeURIComponent(userId)}`)
+				const { data } = await axios.get(generateOcsUrl('apps/user_status/api/v1', 2) + `/statuses/${encodeURIComponent(userId)}`)
```

Comment:
Maybe take advantage of the latest `@nextcloud/router` ?
```suggestion
				const { data } = await axios.get(generateOcsUrl('apps/user_status/api/v1/statuses/{userId}', {
					userId: encodeURIComponent(userId)
				}))
```

Reply:
Ah, yes it does https://github.com/nextcloud/nextcloud-router/blob/7485bfc83f5f170d9050778601ae5881cbb981e0/lib/index.ts#L76